### ISSUE TITLE
runtime namespace becomes undefined after certain sequence of requests

### ISSUE THREAD

**Issue by keganlance** [mozilla/webextension-polyfill#65]
manifest.json:
```
{
  "manifest_version": 2,
  "name": "Test",
  "version": "1.0.0",
  "background": {
    "scripts": [ "browser-polyfill.js", "background.js" ]
  },
  "permissions": [ "storage" ]
}
```

background.js:
```
var someUrl = browser.runtime.getURL("background.js");
var storageResult = browser.storage.local.get();
```

When loading the extension, the background page gives the following warning 11 times:
```
Previous API instantiation failed.
```
and the following error once:
```
(BLESSED_EXTENSION context for oanfllegiohelcgnppacfjannjilkjpm) extensions::lastError:98: Uncaught Error: runtime namespace is null or undefined{Error: runtime namespace is null or undefined
    at assertRuntimeIsAvailable (extensions::lastError:98:11)
    at Object.clear (extensions::lastError:84:3)
    at handleResponse (extensions::sendRequest:77:15)}
```

This error appears every time a request is made to browser.storage. If however, I remove the someUrl line, storage requests keep working fine.

I've tried with both the minified and non minified version of the polyfill. Both using the latest release. I've been able to reproduce this issue in Google Chrome, Chromium and Vivaldi. All tests passed when building both polyfill scripts.

**Comment by rpl** [comment]
@keganlance This is likely to be a dupe of #58:

"Previous API instantiation failed." warning is a regression introduced by #57 (and #64 is the PR with the currently proposed fix).

In the meantime (until we release the final fix for #58), if you don't need the devtools API (which is only wrapped starting by the version 0.2.0), you can use the previous version (0.1.2).

**Comment by Rob--W** [comment]
This has been fixed.
